### PR TITLE
Completed javascript-operators-and-expressions - Tam Nguyen

### PR DESCRIPTION
**Pull request by tam-n**
image

### CODE REVIEW

**Comment by tam-n** [comment]
image

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const width = 3;
+const height = 65;
+const area = width * height;
+console.log('Value of area:', area);
+console.log('Type of area:', typeof area);
+
+const bill = 14;
+const payment = 12;
+const change = bill - payment;
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:9]: Your `payment` should be larger than your `bill`. You'll also need to swap line 9. `payment - bill`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Tam!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**